basic/src/test/java/org/jboss/hal/testsuite/test/configuration/messaging/jmsbridge/JMSBridgeAttributesTestCase.java

```diff
@@ -496,4 +500,41 @@ public void editSubscriptionName() throws Exception {
         }
     }
 
+    @Test
+    public void editSourceCredentialReferenceClearText() throws Exception {
+        page.switchToSourceCredentialReference();
+        elytronChecker.setClearTextCredentialReferenceAndVerify(JMS_BRIDGE_ADDRESS, page.getConfigFragment(), "source-credential-reference");
+    }
+
+    @Test
+    public void editSourceCredentialReferenceStoreReference() throws Exception {
+        page.switchToSourceCredentialReference();
+        elytronChecker.setCredentialStoreCredentialReferenceAndVerify(JMS_BRIDGE_ADDRESS, page.getConfigFragment(), "source-credential-reference");
+    }
+
+    @Test
+    public void editSourceCredentialReferenceIllegalCombination() throws IOException, OperationException {
+        page.switchToSourceCredentialReference();
+        elytronChecker.testIllegalCombinationCredentialReferenceAttributes(JMS_BRIDGE_ADDRESS, page.getConfigFragment(), "source-credential-reference");
+    }
+
+    @Test
+    public void editTargetCredentialReferenceClearText() throws Exception {
+        page.switchToTargetCredentialReference();
+        elytronChecker.setClearTextCredentialReferenceAndVerify(JMS_BRIDGE_ADDRESS, page.getConfigFragment(), "target-credential-reference");
+    }
+
+    @Test
+    public void editTargetCredentialReferenceStoreReference() throws Exception {
+        page.switchToTargetCredentialReference();
+        elytronChecker.setCredentialStoreCredentialReferenceAndVerify(JMS_BRIDGE_ADDRESS, page.getConfigFragment(), "target-credential-reference");
+    }
+
+    @Ignore("Ignored until validation to credential-reference is added.")
```

Comment:
Please add to `@Ignore` description the key of HAL enhancement jira to provide client validation.